controller/api/models.py

```diff
@@ -526,6 +567,21 @@ def logs(self, log_lines=str(settings.LOG_LINES)):
         data = subprocess.check_output(['tail', '-n', log_lines, path])
         return data
 
+    def logs(self, log_lines=str(settings.LOG_LINES)):
+        if self._get_logging_mode() == 'standard':
```

Comment:
Yes, I guess my question is if there is any reason why we shouldn't get the logs from the logger component wither ceph is available or not?  It would make the controller logic more simple.